node/external/transactionAPI/errors.go

```diff
@@ -0,0 +1,15 @@
+package transactionAPI
+
+import "errors"
+
+// ErrTransactionNotFound signals that a transaction was not found
+var ErrTransactionNotFound = errors.New("transaction not found")
+
+// ErrCannotRetrieveTransaction signals that a transaction was not found
```

Comment:
`// ErrCannotRetrieveTransaction signals that a transaction cannot be retrieved`

Reply:
refactored